src/tools/login.js

```diff
@@ -0,0 +1,17 @@
+export function Login() {
```

Comment:
The first letter of a function name should not be capitalized.

Reply:
I am calling Login functin in login therefore I capitalized the first letter.

Reply:
I see. I think I'd prefer you named these functions something like `loginToServer` and `logoutFromServer` instead of capitalization (I'm not sure why that isn't triggering a linter error).

Reply:
ok